### DISCUSSION TITLE
Can not integrate `phantom-wagmi-connector` to the latest version of `wagmi`

### DISCUSSION THREAD

**Discussion by Pelumi527** [phantom/docs#314] [Help & Support]
### Summary

I am trying to integrate `phantom-wagmi-connector` into my dapp uisng wagmi, but it seems the connector is outdated

### Steps to Reproduce

just try to integrate `phantom-wagmi-connector` with the latest release of wagmi 

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by IGMontero** [comment]
Hello, were you able to do this? I'm running into the same issue and IDK if I should just downgrade wagmi?
